### ISSUE TITLE
Whats the deal with copyright?

### ISSUE THREAD

**Issue by Xplodin** [magmafoundation/Magma-1.16.x#230]
You both are open source right?
Which mean you can copy code from forge and paper, and foxserver can copy from magma since its derived from the same copyright.

From what currently understanding is why the BS.
Please let me know the difference between whichever copyright you use vs open source if this is wrong

**Comment by Xplodin** [comment]
Nvm you use LGPL not open source my b

**Comment by Hexeption** [comment]
The issue isn't copying the issue is removing the Copyright notice/Header of our code and that is against the LGPL license

**Comment by Xplodin** [comment]
Gotcha, 
I see what party did wrong now.
Thats a nope in terms of law. Like its there for a reason. It shouldnt be modified and if it is, it cant be in public projects

**Comment by Luohuayu** [comment]
https://git.rbqcloud.cn/Luohuayu/FoxServer
D(D)M(D)C(O)A(S) it, please

**Comment by JustRed23** [comment]
> https://git.rbqcloud.cn/Luohuayu/FoxServer D(D)M(D)C(O)A(S) it, please lmao~~~

Might as well post this here again... [and link the original comment while I'm at it](https://github.com/Luohuayu/FoxServer-Dev/issues/7#issuecomment-1100009357).


The license stated this:

You may convey a Combined Work under terms of your choice that,
taken together, effectively do not restrict modification of the
portions of the Library contained in the Combined Work and reverse
engineering for debugging such modifications, if you also do each of
the following:

a) Give prominent notice with each copy of the Combined Work that
the Library is used in it and that the Library and its use are
covered by this License.

b) Accompany the Combined Work with a copy of the GNU GPL and this license
document.

c) For a Combined Work that displays copyright notices during
execution, include the copyright notice for the Library among
these notices, as well as a reference directing the user to the
copies of the GNU GPL and this license document.

TL;DR you need to include the license, retain the original package names (or add a notice that you changed the package name), and leave all license headers intact. You have changed the LICENSE which means that you did not follow the licensing.

**Comment by Luohuayu** [comment]
> > https://git.rbqcloud.cn/Luohuayu/FoxServer D(D)M(D)C(O)A(S) it, please lmao~~~
> 
> Might as well post this here again... [and link the original comment while I'm at it](https://github.com/Luohuayu/FoxServer-Dev/issues/7#issuecomment-1100009357).
> 
> The license stated this:
> 
> You may convey a Combined Work under terms of your choice that, taken together, effectively do not restrict modification of the portions of the Library contained in the Combined Work and reverse engineering for debugging such modifications, if you also do each of the following:
> 
> a) Give prominent notice with each copy of the Combined Work that the Library is used in it and that the Library and its use are covered by this License.
> 
> b) Accompany the Combined Work with a copy of the GNU GPL and this license document.
> 
> c) For a Combined Work that displays copyright notices during execution, include the copyright notice for the Library among these notices, as well as a reference directing the user to the copies of the GNU GPL and this license document.
> 
> TL;DR you need to include the license, retain the original package names (or add a notice that you changed the package name), and leave all license headers intact. You have changed the LICENSE which means that you did not follow the licensing.

China does not have DMCA, I am waiting for a cross-border lawsuit. lol : )

**Comment by Xplodin** [comment]
Even if they dont have DMCA, everyone should still be civil. Its childish to act like this.

**Comment by JustRed23** [comment]
> Even if they dont have DMCA, everyone should still be civil. Its childish to act like this.

Agreed

**Comment by UeberallGebannt** [comment]
> China does not have DMCA, I am waiting for a cross-border lawsuit. lol : )

Doing illegal things is nothing to be proud of. 
There is no reason to use your software as it is just Magma/Mohist. If you feel alright being a shitty person who isn't even able to create something on their own, go ahead, we can't stop you. You will be known as the guy that isn't able to acomplish anything, congratiolations :)

**Comment by JustRed23** [comment]
> How much of CatServer code was stolen by Magma and Mohist 1.12.2?
Think with your own conscience.

Btw the thing is, mohist also stole from magma... compare the dates of the patches and you'll see

**Comment by Shawiizz** [comment]
> > > China does not have DMCA, I am waiting for a cross-border lawsuit. lol : )
> > 
> > Doing illegal things is nothing to be proud of. There is no reason to use your software as it is just Magma/Mohist. If you feel alright being a shitty person who isn't even able to create something on their own, go ahead, we can't stop you. You will be known as the guy that isn't able to acomplish anything, congratulations :)
> 
> How much of CatServer code was stolen by Magma and Mohist 1.12.2?
> Think with your own conscience.
> ![dog](https://user-images.githubusercontent.com/29095770/163560319-520e63ac-d170-41bb-aa16-52225079e19f.jpg)

How much code? No code.
We are not code stealers, but you are. You break the license while we don't. The only one breaking the rules is you, so you have nothing to say.
You should think before speaking.

**Comment by Xplodin** [comment]
> > How much of CatServer code was stolen by Magma and Mohist 1.12.2?
> > Think with your own conscience.
> 
> Btw the thing is, mohist also stole from magma... compare the dates of the patches and you'll see. Oh how the turntables

Honestly as long as people just leave the code without modifying the license, everyone can go about their day and not have to deal with copyright, dmca, etc.
I personally wouldnt mind if my code was used in a new project by someone online, just dont modify the license. :)
If anything, it means that code was useful and provided help to a community. I wouldnt use the terms stolen unless it was a direct copy-paste project. Some files? nah go ahead and use em. all files? that seems sketchy tbh

Anyways, hope everyone can just get along and keep minecraft hybrids alive. Have a nice day everyone.

**Comment by kingdevnl** [comment]
The main reason is that they just copied the code and as of our license it requires to have the files in the same java package and the magma copyright header must be on the top of the files.
